test_containers/postgres/docker-entrypoint.sh

```
@@ -0,0 +1,359 @@
+			         "docker run".
+			********************************************************************************
+		EOWARN
+	fi
+}
+
+# usage: docker_process_init_files [file [file [...]]]
+#    ie: docker_process_init_files /always-initdb.d/*
+# process initializer files, based on file extensions and permissions
+docker_process_init_files() {
+	# psql here for backwards compatibility "${psql[@]}"
+	psql=( docker_process_sql )
```

Shellcheck (0.10.0)

[warning] psql appears unused. Verify use (or export if used externally).

(SC2034)